bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/CheckBoxImpl.java

```diff
@@ -67,4 +68,17 @@ public Orientation getOrientation() {
         return orientation;
     }
 
+    @Override
+    public boolean isEnforceEnum() {
+        return false;
+    }
+
+    @Override
+    public Object[] getEnums() {
```

Comment:
Why is this not extending from AbstractOptionsField ? Ideally it should. Also, why is enforceEnum always false